### ISSUE TITLE
Fail in before hook in structured test

### ISSUE THREAD

**Issue by Paprikh7** [mmisty/cypress-allure-adapter#95]
@Hi,
I try to write more organized tests. However, I encountered a problem when the reporter does not log the setup when it falls in before. Here is an example of code that I am able to replicate this on.
```
describe("test fail in before", function() {
    describe('test fail 2 before', function() {
        before(function() {
            cy.visit('/')
            cy.get('notexistent').click()
        })
        it("Login as user",function () {
            cy.visit('/')
        });
        it("Login as admin",function () {
            cy.visit('/')
        });
    })
})
```
<img width="2050" alt="Screenshot 2023-12-21 at 14 05 46" src="https://github.com/mmisty/cypress-allure-adapter/assets/109957805/6dee860a-27ae-49a9-84ec-6587a4d9df8f">
confing env:

```
env: { 
            "allure": "true",
            "allureResults": "../../../allure-results",
            "allureAddVideoOnPass": "true",
            "allureSkipCommands": 'intercept,then,testParameters,wrap,wait',
            "allureSkipSteps": '"after each" hook*,"before each" hook*,"after all",ignore-allure-log',
        },
```

**Comment by mmisty** [comment]
Hi @Paprikh7 
Did you run these tests in interactive mode or in run mode?
If interactive, could you please confirm that the same happens in run mode?
thank you

**Comment by Paprikh7** [comment]
It was run mode
Thank you

**Comment by mmisty** [comment]
@Paprikh7 could you please confirm that the same happens when you use 1 'describe' instead of 2 nested ones?

by the way - if you use second describe just to have better structure in your the report you can use allure interface to achieve that instead of creating more describes. let me know if you need more info.

**Comment by Paprikh7** [comment]
It won't happen, when I use 1 'describe'.
It happen only use 2 or more 'describe'.

**Comment by mmisty** [comment]
should be fixed in v2.0.10
